.github/workflows/index-resources.yml

```diff
@@ -34,6 +119,18 @@ jobs:
           node resourceIndexer/main.js \
             --gzip --output resources.json.gz \
             --resourceTypes dataset --collections core staging
+      - name: Extract RESOURCE_INDEX from config
+        env:
+          RESOURCE_INDEX_PREFIX: ${{ inputs.resource_index_prefix }}
+        run: |
+          if [[ -f "./get-resource-index" ]]; then
```

Comment:
(Just skimming this PR - will do a proper review later) - this conditional is just to allow the action to work while the canary/production servers are still using versions of the code prior to the introduction of `get-resource-index`?

Reply:
Yup, I'll add a comment.